multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/zeistmanga/ZeistMangaDto.kt

```diff
@@ -0,0 +1,75 @@
+package eu.kanade.tachiyomi.multisrc.zeistmanga
+
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import org.jsoup.Jsoup
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+@Serializable
+data class ZeistMangaDto(
+    @SerialName("feed") val feed: ZeistMangaFeedDto? = null,
```

Comment:
These attributes with the `@SerialName` that are exactly equal to the attribute name in the class can be removed.

Take this as an example:

```suggestion
    val feed: ZeistMangaFeedDto? = null,
```